src/sensors/distance/infrared/analog/sharp/GP2Y0A02.cpp

```diff
@@ -16,9 +16,7 @@ GP2Y0A02::GP2Y0A02(Runtime& runtime, uint8_t pin)
 unsigned int GP2Y0A02::getDistance()
 {
     auto analogReading = mRuntime.getAnalogPinState(kPin);
-    // It's OK to surpress this clang-tidy warning since this is part of a magic (!) formula
-    // NOLINTNEXTLINE(readability-magic-numbers)
-    auto result = static_cast<unsigned int>(9462 / (analogReading - 16.92));
+    auto result        = static_cast<unsigned int>(9462 / (analogReading - 16.92));
```

Comment:
:warning: **readability-magic-numbers** :warning:
9462 is a magic number; consider replacing it with a named constant
```suggestion
    auto result        = static_cast<unsigned int>(9462 / (analogReading - 16.92));

```